src/platform/plugins/shared/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -622,6 +622,7 @@ export class SearchInterceptor {
     this.deps.toasts.addDanger({
       title: 'Timed out',
       text: toMountPoint(e.getErrorMessage(this.application), this.startRenderServices),
+      toastLifeTimeMs: 1000 * 60 * 60,
```

Comment:
```suggestion
      toastLifeTimeMs: Infinity,
```
It might be best in this case to just set the timeout to `Infinity` so the toast is there no matter how long before the user returns.

Reply:
It was my first approach, however the toast behaves differently then, I'm not sure if it's desired. It disappears after hover (on mouse leave), while numeric lifetime keeps it open until this time passes or you close it manually. Below a quick video with `Infinity`, let me know which version we prefer (and also maybe it's a bug we want to investigate?)

https://github.com/user-attachments/assets/a53dc342-1e5f-4298-8e48-80837b50789e

Reply:
To compare, video with `1000 * 60 * 60`:
https://github.com/user-attachments/assets/f1d22413-2237-4a4c-9441-13d851343c19

Reply:
To Infinity and beyond! I agree that this would be the ideal behavior, but I also assume, to go there, it will be a rabbit hole, since toasts API need to learn the concept of Infinity ... So let's keep  infinity out of scope. Compared to before 1h of toast is infinity ... we could multiply by 24, for those coming back the next day to see what are the results ... but 24 hours toast, is it still tasty?

Reply:
I heard toasts are quite packed with preservatives, let's test if it's still fresh after a week (as @davismcphee suggested during an offline discussion) 🍞 
Go hard or go home 😁

Reply:
Thanks for updating it @akowalska622! Since `Infinity` is buggy, I think a week should definitely be sufficient. That's well beyond when toast should be consumed anyway 😄

We discussed this in our sync earlier, but for context, EUI does [explicitly support](https://github.com/elastic/eui/blob/0e31d35b61da243be0167b6b62748ae0baf56377/packages/eui/src/services/time/timer.ts#L19) `Infinity` for toast lifetimes (made sure to check before suggesting), but as @akowalska622 pointed out, there seems to be a bug currently. Not something for us to investigate for now, so a week it is 👍